packages/overdrive/lib/components/TextLink/TextLink.tsx

```diff
@@ -14,7 +14,7 @@ import {
 } from 'react';
 import { useStyles } from 'react-treat';
 
```

Comment:
Dont know here to put this comment..

But the width is too wide, could just be the story??

![image](https://user-images.githubusercontent.com/599459/88248503-bc2e9280-cce4-11ea-8fbb-73d8e0b46d77.png)